src/NuGet.Clients/NuGet.CommandLine/Program.cs

```diff
@@ -68,6 +69,8 @@ public static int MainCore(string workingDirectory, string[] args)
             }
 
             var console = new Common.Console();
+            Logger.Instance = console;
```

Comment:
Thanks for the clarification @alpaix, makes sense.
I'll add a //todo in this PR and log a separate issue to track this.